### PR TITLE
Support IPv6

### PR DESCRIPTION
**Pull request by swiftgist**
This reverts commit 7fc208925ad4ba95117ac06dea6ad6c6ddcf6f6a.

Signed-off-by: Eric Jackson <swiftgist@gmail.com>
jira: SES-119
-----------------

**Checklist:**
- [x] Added unittests and or functional tests
- [ ] Adapted documentation
- [x] Referenced issues or internal bugtracker
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by swiftgist** [comment]
@susebot run teuthology

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier1

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier2/rgw

**Comment by smithfarm** [comment]
@susebot run teuthology

File: srv/modules/runners/net.py [hunk 1]
```diff
@@ -107,11 +109,19 @@ def iperf(cluster=None, exclude=None, output=None, **kwargs):
...
+            if 'public_network' in public_networks[host]:
+                ipversion = _ipversion(public_networks[host]['public_network'])
+                break
+
         log.debug("iperf: public_network {} ".format(public_networks))
         cluster_networks = local.cmd(search, 'pillar.item',
                                      ['cluster_network'], tgt_type="compound")
         log.debug("iperf: cluster_network {} ".format(cluster_networks))
-        total = local.cmd(search, 'grains.get', ['ipv4'], tgt_type="compound")
+        total = local.cmd(search, 'grains.get', [ipversion], tgt_type="compound")
```
Comment (jschmid1) [inline srv/modules/runners/net.py]: Mh, I don't find `ipversion` in my grains. Is that something we write?

Comment (jschmid1) [reply srv/modules/runners/net.py]: respectively ipv6/ipv4 I mean by that..

Comment (jschmid1) [reply srv/modules/runners/net.py]: yikes, looked at the wrong data.. all there

**Pull request merged**